Gifski/VideoValidator.swift

```diff
@@ -93,6 +96,25 @@ struct VideoValidator {
 			return .failure
 		}
 
+		// If the video track duration is shorter than the total asset duration, we extract the video track into a new asset to prevent problems later on.
+		guard firstVideoTrack.isFullDuration else {
```

Comment:
do we still need to do this after you changed the code to base on the track instead of the asset?

Reply:
Yes, because the part of the video where there's no video track will still show in the trimmer view if not, which might confuse users, and also if the user trims it to only that part, then the conversion will start, but nothing will happen as there's nothing to convert.

Reply:
I've expanded the comment about it: https://github.com/sindresorhus/Gifski/pull/165/commits/ebe1e7e58c01127ddb46467d1b6a381f0194ef11